integration-tests/src/test/java/uk/gov/di/authentication/api/ProcessingIdentityIntegrationTest.java

```diff
@@ -138,7 +139,34 @@ void shouldReturnStatusOfERRORWhenEntryIsNotInDatabase() throws Json.JsonExcepti
                                 new ProcessingIdentityResponse(ProcessingIdentityStatus.ERROR))));
     }
 
-    private void setupSession() throws Json.JsonException {
+    @Test
+    void shouldReturnStatusOfNO_ENTRYWhenEntryIsNotInDatabaseOnFirstAttempt()
+            throws Json.JsonException {
+        setupSession(false);
+        setupClient();
+        setupUser();
+
+        Map<String, String> headers = new HashMap<>();
+        headers.put("Session-Id", SESSION_ID);
+        headers.put("Client-Session-Id", CLIENT_SESSION_ID);
+        headers.put("X-API-Key", FRONTEND_API_KEY);
+
+        var response =
+                makeRequest(
+                        Optional.of(format("{ \"email\": \"%s\"}", TEST_EMAIL_ADDRESS)),
+                        headers,
+                        Map.of());
+
+        assertThat(response, hasStatus(200));
+        assertThat(
+                response,
+                hasBody(
```

Comment:
Very minor thing - we have a `hasJsonBody` matcher that takes an object that we could use here instead.

Reply:
Good shout. Updated.